.github/scripts/no_pandas.py

```diff
@@ -0,0 +1,72 @@
+# %%
```

Comment:
This script runs in the code-check CI and does a static check that pandas imports aren't used.

see https://github.com/posit-dev/great-tables/pull/261/files#diff-38696fb127ded399e74cd12e7529270058aa86df58256e55b02c5b382adf908aR31